void's qt6 is patched[1] to not use any of the *64 functions and types
because they are useless/deprecated. Since there is no such define for
the fstatat function, kio has their own define that needs to be patched
out to avoid using `fstatat64` wth `struct stat`.

[1]: srcpkgs/qt6-base/patches/musl-lfs.patch

--- a/src/kioworkers/file/file_unix.cpp
+++ b/src/kioworkers/file/file_unix.cpp
@@ -1115,13 +1115,7 @@
     return WorkerResult::fail(KIO::ERR_CANNOT_SYMLINK, dest);
 }
 
-// Qt maps QT_STATBUF to struct stat64 under large file support, and there is no QT_FSTATAT, so the
-// same choice is made here for the directory-relative fstatat().
-#if defined(QT_USE_XOPEN_LFS_EXTENSIONS) && defined(QT_LARGEFILE_SUPPORT)
-#define FSTATAT ::fstatat64
-#else
 #define FSTATAT ::fstatat
-#endif
 
 WorkerResult FileProtocol::deleteUnder(int dfd, KIO::filesize_t &removed)
 {
